src/markdown-pages/build-apps/nr1-deprecations.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Manage New Relic One SDK deprecations'
+template: 'GuideTemplate'
+description: 'Learn how to manage New Relic One SDK deprecations'
+---
+
+We at New Relic frequently update the New Relic One SDK. These updates include new features, improvements to existing features, and the deprecation, or removal, of existing features. When we remove a feature, you, as a developer of New Relic One Nerdpacks, need to know because if you don't update your code, it will eventually stop working.
```

Comment:
```suggestion
At New Relic, we frequently update the New Relic One SDK. These updates include new features, improvements to existing features, and the deprecation, or removal, of old features. When we improve, or remove a feature, you, as a developer of New Relic One Nerdpacks, need to be aware of these changes because if you don't update your code, it may eventually stop working.
```